### PR TITLE
Improve plan template

### PR DESCRIPTION
**Pull request by weihesdlegend**
## Description
* Autofill start time when adding new rows
* Add location autofill button
* Add price range selection dropdown

## Testing
- [x] Integration testing on Heroku staging <- Deployed to https://staging-vp.herokuapp.com/v1/template
- [ ] Added new unit tests

## Checks
- [x] Have you removed commented code?
- [x] Have you used gofmt to format your code?

### CODE REVIEW

File: assets/js/plan_template.js [hunk 1]
```diff
@@ -27,10 +33,14 @@ function insertNewRow(start = '8', end = '10', category = 'Visit') {
     let cafeOption = document.createElement("option");
     let attractionOption = document.createElement("option");
 
-    let startTime = hourDropdown();
+    if (template.rows.length > 2) {
+        start = lastColumnOfLastRow.firstChild.value;
+    }
+
+    let startTime = hourDropdown(start);
```
[the remark this answers is not in the document]
Comment (weihesdlegend) [reply assets/js/plan_template.js:40]: In this case, would it make sense to just fill the end time with start time?

[the remark this answers is not in the document]
Comment (rwangsc18) [reply assets/js/plan_template.js:40]: This case happens when I clicked add twice in a row. The first new row is fine, which has startTime 18, and no endTime (waiting for user input). But the 2nd new row is a bit strange with a default endTime equal to 10. Maybe we can just leave the startTime empty, if the previous entry doesn't have an endTime.

[the remark this answers is not in the document]
Comment (weihesdlegend) [reply assets/js/plan_template.js:40]: Done.

[the remark this answers is not in the document]
Comment (rwangsc18) [reply assets/js/plan_template.js:40]: Tested myself. LGTM

**Review by rwangsc18 — APPROVED** [review]
LGTM

**Pull request merged**